src/mtx/CommandBinaryBuffer.ts

```diff
@@ -351,6 +357,20 @@ class CommandBinaryBuffer extends BinaryBuffer {
         super(dataOrLength, false);
     }
 
+    getFrameHeader (): IFrameHeader {
```

Comment:
for future use?

Reply:
![image](https://github.com/jooby-dev/jooby-codec/assets/1518095/20139e7d-8a9f-4591-8aab-c26f76801ffd)
see in src/mtx/message.ts